Commands/GenerateWebpImages.php

```diff
@@ -27,6 +28,15 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        /** @var WebpEncoderFactory $encoderFactory */
+        $encoderFactory = $this->container->get('shyim_webp.services.webp_encoder_factory');
+        $runnableEncoders = WebpEncoderFactory::onlyRunnable($encoderFactory->getEncoders());
```

Comment:
I would do suround it with current, so you can get rid of [0]

Reply:
True, `current` is key independant. The 0 key is ensured by `array_values` from `WebpEncoderFactory::onlyRunnable`